server/models/user_model.py

```diff
@@ -65,3 +46,64 @@ def update_experience(cls, id, exp):
         user = cls.query.get(id)
         user.experience = exp
         db.session.commit()
+
+    @classmethod 
+    def search_experience(cls,lang_levels, reviewee_id):
+        # search users the are above certain experience level, return list of 
+        # users that match the requirements
+        req_language = list(lang_levels)[0]
+        req_level = int(lang_levels[req_language])
+        users = cls.query.all()
```

Comment:
Ideally like I mentioned we do a query in the future, once we redesign experience level

Reply:
We could at least for now do a query where id is not the reviewee_id and user.experience is not null